crs_application/src/task_managers/part_registration_manager.cpp

```diff
@@ -77,7 +101,19 @@ common::ActionResult PartRegistrationManager::setInput(const datatypes::ScanAcqu
 
 common::ActionResult PartRegistrationManager::computeTransform()
 {
-  RCLCPP_WARN(node_->get_logger(),"%s not implemented yet",__PRETTY_FUNCTION__);
+  // todo(ayoungs) get vector of point clouds from scan_acquisition_manager
+  auto localize_to_part_request = std::make_shared<crs_msgs::srv::LocalizeToPart::Request>();
+  localize_to_part_request->point_clouds = std::vector<sensor_msgs::msg::PointCloud2>();
+
+  auto localize_result_future = localize_to_part_client_->async_send_request(localize_to_part_request);
+  if (rclcpp::spin_until_future_complete(node_, localize_result_future) !=
+      rclcpp::executor::FutureReturnCode::SUCCESS)
+  {
+      RCLCPP_ERROR(node_->get_logger(), "Localize to Part service call failed");
+      return false;
+  }
+  auto localize_result = localize_result_future.get();
+  // todo(ayoungs) save off transform
```

Comment:
@agyoungs will you be addressing this TODO in this PR?

Reply:
yes